src/coreComponents/linearAlgebra/solvers/GmresSolver.cpp

```diff
@@ -70,42 +67,39 @@ void ApplyGivensRotation( real64 const c, real64 const s, real64 & dx, real64 &
   dx = temp;
 }
 
-void Backsolve( localIndex const k,
+void Backsolve( integer const k,
                 arraySlice2d< real64 const, MatrixLayout::COL_MAJOR > const & H,
                 arraySlice1d< real64 > const & g )
 {
-  for( localIndex j = k - 1; j >= 0; --j )
+  for( integer j = k - 1; j >= 0; --j )
```

Comment:
this is not a `localIndex`?

Reply:
No, here `k` (and by extension `j` and `i`) varies between 0 and the size of Krylov subspace (i.e. max number of restart iterations), which is on the order of ~100. I've recently been replacing many uses of `localIndex` that I find incorrect, i.e. whenever a quantity does not scale with the size of local mesh partition.